src/mesh/Mesh.cpp

```diff
@@ -277,7 +277,7 @@ void Mesh::computeBoundingBox()
   PRECICE_TRACE(_name);
 
   // Keep the bounding box if set via the API function.
-  BoundingBox bb = _boundingBox.empty() ? BoundingBox(_dimensions) : BoundingBox(_boundingBox);
+  BoundingBox bb = _boundingBox.isDefault() ? BoundingBox(_dimensions) : BoundingBox(_boundingBox);
```

Comment:
I do not understand this change (it may be correct, it may be wrong). Could you please explain?

Reply:
The code change itself is only a renaming of the function. We have to preserve the bounding box in case there is reasonable data in there.